src/client/src/rt-platforms/openfin/components/index.ts

```diff
@@ -3,3 +3,4 @@ export { OpenFinLogo } from './OpenFinLogo'
 export { default as OpenFinBrowserLink } from './OpenFinBrowserLink'
 export { OpenFinWindowFrame } from './OpenFinWindowFrame'
 export { OpenFinSubWindowFrame } from './OpenFinSubWindowFrame'
+export { default as ToggleOpenFinLayoutLock } from './ToggleOpenFinLayoutLock'
```

Comment:
Ah, I missed that this is being exported and rendered in `Header`. That's a leaky abstraction; is it possible for this to be rendered inside `OpenFinSubWindowFrame` or will that involve too big a refactor? At a minimum we should mark this as tech debt and create a ticket.